pkgs/servers/nextcloud/packages/29.json

```diff
@@ -91,15 +91,15 @@
     "homepage": ""
   },
   "mail": {
-    "hash": "sha256-RWOnUSJBDwSajFafYYv3NnHI93g0lRPKaRU3/Vf8qy8=",
-    "url": "https://github.com/nextcloud-releases/mail/releases/download/v3.7.2/mail-v3.7.2.tar.gz",
-    "version": "3.7.2",
+    "hash": "sha256-zLKTXqXTKyCRrETjKTJzO84QdYar/4zjUrbvT0aWfx0=",
+    "url": "https://github.com/nextcloud-releases/mail/releases/download/v3.7.5/mail-v3.7.5.tar.gz",
+    "version": "3.7.5",
     "description": "**💌 A mail app for Nextcloud**\n\n- **🚀 Integration with other Nextcloud apps!** Currently Contacts, Calendar & Files – more to come.\n- **📥 Multiple mail accounts!** Personal and company account? No problem, and a nice unified inbox. Connect any IMAP account.\n- **🔒 Send & receive encrypted mails!** Using the great [Mailvelope](https://mailvelope.com) browser extension.\n- **🙈 We’re not reinventing the wheel!** Based on the great [Horde](https://horde.org) libraries.\n- **📬 Want to host your own mail server?** We do not have to reimplement this as you could set up [Mail-in-a-Box](https://mailinabox.email)!\n\n## Ethical AI Rating\n\n### Priority Inbox\n\nPositive:\n* The software for training and inferencing of this model is open source.\n* The model is created and trained on-premises based on the user's own data.\n* The training data is accessible to the user, making it possible to check or correct for bias or optimise the performance and CO2 usage.\n\n### Thread Summaries (opt-in)\n\n**Rating:** 🟢/🟡/🟠/🔴\n\nThe rating depends on the installed text processing backend. See [the rating overview](https://docs.nextcloud.com/server/latest/admin_manual/ai/index.html) for details.\n\nLearn more about the Nextcloud Ethical AI Rating [in our blog](https://nextcloud.com/blog/nextcloud-ethical-ai-rating/).",
     "homepage": "https://github.com/nextcloud/mail#readme"
   },
   "maps": {
-    "hash": "sha256-Qkx4TqZf0ygVNLl6NtyOBiW0DrUlxQlStk/SBw2w9KE=",
-    "url": "https://github.com/nextcloud/maps/releases/download/v1.4.0-1-nightly/maps-1.4.0-1-nightly.tar.gz",
+    "hash": "sha256-LOQBR3LM7artg9PyD8JFVO/FKVnitALDulXelvPQFb8=",
+    "url": "https://github.com/nextcloud/maps/releases/download/v1.4.0/maps-1.4.0.tar.gz",
```

Comment:
To be fair, it got updated back to a stable version here, but the issue persists with contacts.
And it will appear again if we don't find a better solution.

Reply:
That happened so many times in the past. No reason raise the issue right now.

Reply:
Left a reply in the other thread.
I do see an issue in the new fetcher, but that's indeed a false-positive.